bin/s2nc.c

```diff
@@ -158,7 +158,7 @@ static void setup_s2n_config(struct s2n_config *config, const char *cipher_prefs
                     fprintf(stderr, "Error allocating memory\n");
                     exit(1);
                 }
-                memcpy(protocols[idx], next, length);
+                memcpy_check(protocols[idx], next, length);
```

Comment:
You can't memcpy_check in a void function.

Reply:
Similar to GUARD_EXIT(), would a `memcpy_exit()` be appropriate for the `bin/` tools? It would be defined in the `bin/common.h` file. I think s2nc and s2nd should still perform appropriate memory checks, but I wouldn't want a complex regular expression in the grep_simple_mistakes script just to handle the s2nc and s2nd files.